guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -3,6 +3,8 @@
 *********************************************
 Installing SQream with Monit
```

Comment:
I don't think "Installing" is the right term here. Perhaps we should name it "Launching"